__tests__/4-get-api-reviews.test.js

```diff
@@ -0,0 +1,63 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed")
+const connection = require("../db/connection");
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    return connection.end()
+})
+
+describe('testing the get api/reviews/:review_id endpoint', () => {
+    test('testing that get api/reviews/:review_id returns the right information and returns status 200', () => {
+        return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((result) => {
+          const reviewObj = result.body
+            
+            expect(reviewObj).toHaveProperty('review')
```

Comment:
I'd remove lines 23 - 32 as you are using a flexible matcher in the test below (lines 34- 43) this is sufficient 😄